inst/stan/data/simulation_delays.stan

```diff
@@ -1,5 +1,11 @@
-  int delays;                           // no. of delay distributions
-  real delay_mean[n, delays];           // mean of delays
-  real delay_sd[n, delays];             // sd of delays
-  int delay_max[delays];                // maximum delay
-  int delay_dist[delays];                // 0 = lognormal, 1 = gamma
+  int<lower = 0> delay_n;                  // number of delay distribution distributions
```

Comment:
we could consider moving to a chunk based model for some of these repeated scripts to avoid all this copy and pasting - comes at the cost of model readability. Could also make a simple generator that we run to build the stan code but this would easily lead to dev problems I think (with things getting out of sync).

Reply:
I agree, still quite a repetition. Will put in a separate issue.